src/couch/src/couch_server.erl

```diff
@@ -937,6 +938,14 @@ num_servers() ->
     erlang:system_info(schedulers).
 
 
+aggregate_queue_len() ->
+    N = couch_server:num_servers(),
```

Comment:
Are you using the fully qualified external call on purpose for reasons I'm not thinking of?

Reply:
erm, nope. In the first commit it was in chttpd_node